deps.edn

```diff
@@ -56,4 +64,9 @@
   {:extra-deps {cloverage/cloverage {:mvn/version "1.2.2"}}
    :main-opts  ["-m" "cloverage.coverage"
                 "-p" "src/main"
-                "-s" "src/test"]}}}
+                "-s" "src/test"]}
+  :build
+  {:paths      ["src/build"]
+   :deps       {io.github.clojure/tools.build {:git/tag "v0.6.6"
```

Comment:
Glad to see `tools.build` is used here, this is the way.

Reply:
This might work just as well with `:replace-deps` rather than `:deps`, I believe that is what is suggested in the documentation. A nice way to ensure the purity of the build environment

Reply:
I thought plain `:deps` would do that, but it seems like `:replace-deps` is closer to my intentions regardless.

Reply:
It's worth checking, it's possible they are equivalent. `tools.deps` has evolved pretty quickly.

Reply:
Regardless I changed `:deps` to `:replace-deps` and :paths` to :replace-paths` since those are the keys you're "supposed" to use: https://github.com/yetanalytics/project-persephone/pull/39/commits/e0726ac0fbe8c54ee0544fa7adb9c9407d05dda3